src/util/privkey.rs

```diff
@@ -139,34 +144,46 @@ impl FromStr for Privkey {
             network: network,
             key: key
         })
+	}
+}
+
+impl fmt::Display for Privkey {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+		self.fmt_wif(fmt)
```

Comment:
spacing nit

Reply:
Not sure what you mean here? You want it to be on one line or so?

Reply:
Why are there 12 spaces instead of 8?

Reply:
Oh, it's a tab. Please do not use tabs.

Reply:
requiring rustfmt on all PRs should remove the necessity to have this kind of discussion

Reply:
Yeah, a rustfmt file would help here :) Or at least a `.editorconfig` so that I can turn off rustfmt and not make spaces. I have to confess... I'm a [tab man](https://www.youtube.com/watch?v=iSphnRYhCvE).

Reply:
I converted all tabs into spaces.